### PR TITLE
Add full page example to error summary component

### PR DESCRIPTION
**Pull request by dashouse**
Add full page example to error summary component to help users understand where it should be positioned and what size column it should be in.

Relates to issues:
https://github.com/alphagov/govuk-design-system/issues/851
https://github.com/alphagov/govuk-design-system/issues/860

This example has placeholder content. @amyhupe can we talk about how we might include "component context" in the content pattern as it feels like we might need to do this more often.

For other observers, this is currently low priority. No need to look at it w/c 13/05/19.

### CODE REVIEW

File: src/components/error-summary/index.md.njk [hunk 1]
```diff
@@ -54,6 +54,12 @@ For questions that require a user to select one or more options from a list usin
 
 {{ example({group: "components", item: "error-summary", example: "linking-checkboxes-radios", html: true, nunjucks: true, open: false, size: "s"}) }}
 
+### Example in context
+
+The error summary should sit at the top of the `main` container. If you page has breadcrumbs or a back link it should be below these, but before the `<h1>`.
```
Comment (nickcolley) [inline src/components/error-summary/index.md.njk]:
```suggestion
The error summary should sit at the top of the `main` container. If your page has breadcrumbs or a back link it should be below these, but before the `<h1>`.
```

Comment (dashouse) [reply src/components/error-summary/index.md.njk]: Thanks for spotting

File: src/components/error-summary/index.md.njk [hunk 2]
```diff
@@ -54,6 +54,12 @@ For questions that require a user to select one or more options from a list usin
 
 {{ example({group: "components", item: "error-summary", example: "linking-checkboxes-radios", html: true, nunjucks: true, open: false, size: "s"}) }}
 
+### Example in context
```
Comment (amyhupe) [inline src/components/error-summary/index.md.njk]:
```suggestion
### Where to put the error summary
```

File: src/components/error-summary/index.md.njk [hunk 3]
```diff
@@ -54,6 +54,12 @@ For questions that require a user to select one or more options from a list usin
...
+
+The error summary should sit at the top of the `main` container. If your page has breadcrumbs or a back link it should be below these, but before the `<h1>`.
```
Comment (amyhupe) [inline src/components/error-summary/index.md.njk]:
```suggestion
Put the error summary at the top of the `main` container. If your page includes breadcrumbs or a back link, place it below these, but above the `<h1>`.
```

**Comment by dashouse** [comment]
Merged thanks 👍

**Pull request merged**